site/payment-gateway/v2/pay.md

```diff
@@ -44,6 +44,9 @@ NB: All parameter names are case sensitive.
  | paymentReference | String | [A-Z] | Y | A reference used to group multiple payments together, doesn't need to be unique. |
  | amount | Integer | [0-9] | Y | The price (in pence) to be paid. |
  | currency | String | [A-Z]{3} | Y | ISO 4217 standard currency code for the currency of this payment |
+ | postalCode | String | [A-Z0-9] | N | The postal code for the payment's billing address. |
```

Comment:
Can we have a couple of examples for the newly added fields?

Also, at the top of this .md, it would be great to add a note on why we added the postalCode/countryCode(something like `natwest has started refusing payments that dont have this information`)